src/kbmod_wf/task_impls/uri_to_ic.py

```diff
@@ -82,7 +83,13 @@ def uri_to_ic(
 
     logger.info("Creating ImageCollection")
     # Create an ImageCollection object from the list of URIs
-    ic = ImageCollection.fromTargets(uris)
+    last_time = time.time()
+    logger.info("Creating butler instance")
+    this_butler = Butler(runtime_config.get("butler_config_filepath", None))
+    elapsed = round(time.time() - last_time, 1)
+    logger.debug(f"Required {elapsed}[s] to instantiate butler.")
+
+    ic = ImageCollection.fromTargets(uris, butler=this_butler)
```

Comment:
Quick question: I had assumed this will resolve to the fits standardizer rather than the butler standardizer? If so, I would probably just remove using the butler in this task

Reply:
I'm ok removing the butler here. For the time being, we should assume that any uris file is just a list of filename, and not a list of butler UUIDs, yeah?

Reply:
The Image collection is the list of butler UUIDs so I'd say this makes sense. But the workflow wouldn't be scientifically valid if the ButlerStandardizer isn't used because the KBMODV1 standardizer (the way KBMOD used to run) doesn't correct for the zeropoint offsets.

I think they want to run the old thing with the zeropoint correction? Perhaps we should just create an image collection out of those lst files ourselves and dump this.

Reply:
I personally think that's fine.

For now, it's nice to be able to easily rerun the workflows that created the TNOs 2024 results as a reference point (and because it's something we presented) even though we know they won't have the zeropoint fix.

Converting the old multi-night ImageCollections to ButlerStandardizer-created ImageCollections with the zeropoint correction is something we don't have to solve with this PR

Reply:
"Perhaps we should just create an image collection out of those lst files ourselves and dump this."

Yeah, I think this is the approach I've decided on, but I still think it's worthwhile to keep a workflow for re-generating the old results we reported around for now.

Reply:
Looks like Dino has a comment about this as well that I didn't notice until. Leaving this conversation open until I establish the path forward.

Reply:
I think Wilson has a point about leaving it in. We know what we need to do to improve it, but given we held a talk on it we should legacy it. I'm fine with leaving it in for now, knowing it's not correct, then tagging a version and removing it later.

Reply:
Thanks for the discussion here. It sounds like the path forward is to leave this as is. And if we need to reproduce the results of the TNO talk, then we'll make a separate tagged branch that doesn't use the ButlerStandardizer.

Reply:
I spoke with @wilsonbb after this message. Given that we currently have two use cases

1. The pre-TNO workflow that used the FitStandarizer
2. The ImageCollection set that Dino has put together that uses the ButlerStandarizer, but doesn't make use of uri_to_ic

So removing the butler from uri_to_ic should allow for support of both use cases for the time being. 

Everyone please tell me if I'm still not understanding this correctly. 😄

Reply:
Yes, leave the old workflow using FitsStdandardizer even though we know we can improve.